README.md

```diff
@@ -13,13 +13,27 @@ $ npm install koa-stream
 
 ## Usage
 
+### Stream Files
+
+```javascript
+var streamFile = require('koa-stream').streamFile;
+var path   = require('path');
+var app    = require('koa');
+
+app.use(function *() {
+    yield streamFile(this, 'my-video.mp4', {root: path.join(__dirname, '/public')});
+});
+```
+
+### Stream Buffers
+
 ```javascript
-var stream = require('koa-stream');
+var streamBuffer = require('koa-stream').streamBuffer;
 var path   = require('path');
 var app    = require('koa');
 
 app.use(function *() {
-    yield stream('my-video.mp4', {root: path.join(__dirname, '/public')});
+    streamBuffer(this, new Buffer([1,2,3]), 'image/png', {allowDownload: true});
```

Comment:
We can't use yield here because streamBuffer is not a generator function. Koa middlewares somehow also work without a yield statement :)